tests/aruba/aruba.md

```diff
@@ -0,0 +1,50 @@
+# Aruba-rspec testframework for CB CLI
+The aruba test project is located in tests/aruba/
+
+## E2E tests
+
+Run without Docker:
+```
+curl -sSL https://get.rvm.io | bash -s stable --ruby
+source /Users/mhalmy/.rvm/scripts/rvm
+gem install rspec
+gem install aruba
+gem install aruba-rspec
+gem install json
+gem install rspec-json_expectations
+gem install rspec_junit_formatter
+#Edit localvars
+source localvars
+
+rspec spec/e2e/cluster.rb  # Run a specific test
+rspec spec/e2e/*.rb # Run all tests
+rspec -f RspecJunitFormatter -o test-result.xml -f h spec/e2e/*.rb | tee test-result.html | ruby -n # Run with output files
+```
```

Comment:
For this user need to stand in the ...`cb-cli/tests/aruba` path.

Unfortunately I got errors:
```
rspec spec/e2e/credential.rb

An error occurred while loading ./spec/e2e/credential.rb.
Failure/Error: require "oauth"

LoadError:
  cannot load such file -- oauth
# ./spec/e2e/spec_helper.rb:6:in `<top (required)>'
# ./spec/e2e/credential.rb:3:in `<top (required)>'
No examples found.


Finished in 0.00021 seconds (files took 0.23694 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```
```
 rspec spec/e2e/cluster.rb

An error occurred while loading ./spec/e2e/cluster.rb.
Failure/Error: require "oauth"

LoadError:
  cannot load such file -- oauth
# ./spec/e2e/spec_helper.rb:6:in `<top (required)>'
# ./spec/e2e/cluster.rb:4:in `<top (required)>'
No examples found.


Finished in 0.00024 seconds (files took 0.22455 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```